src/chrome-launcher.ts

```diff
@@ -356,6 +356,12 @@ class Launcher {
         };
       }
 
+      // handle process closed right after it started
+      this.chromeProcess.on('close', () => {
```

Comment:
this change fails some of the tests. not sure why at the moment.

But it may need to move slightly.